optimum/intel/openvino/modeling_decoder.py

```diff
@@ -238,10 +266,14 @@ def _from_transformers(
             local_files_only=local_files_only,
             force_download=force_download,
             trust_remote_code=trust_remote_code,
-            model_kwargs=kwargs,
             int8=load_in_8bit,
         )
 
+        # Unpatch modules after GPTQ export
+        if do_gptq_patching:
+            torch.cuda.is_available = orig_cuda_check
+            GPTQQuantizer.post_init_model = orig_post_init_model
+
```

Comment:
I think it could make sense to add a test as well to verify we don't break anything in the future

Reply:
I can add a test that checks that AutoModelForCausalLM can be instantiated without GPU. What do you think?

Reply:
We can add a test for GPTQ quantized model export + inference once the PR is merged in AutoGPTQ then

Reply:
My bad, it was clearly explained in your other comment. Let's merge, we can add a test once enabled in the next openvino release